pepper-apis/src/main/resources/logback.xml

```diff
@@ -17,25 +17,8 @@
         </filter>
     </appender>
 
-    <appender name="SECURE" class="ch.qos.logback.core.rolling.RollingFileAppender">
```

Comment:
I'm good with this--was always a bit of a thought that never really made it out of the gate.

Reply:
Just to clarify: I copied the removal of this appender from your original PR, though I do not understand the reasoning. Sounds like you still want to do this.

Reply:
Well I'm glad I'm consistent with myself :)